jsHelper/spicetifyWrapper.js

```diff
@@ -1501,23 +1510,41 @@ Spicetify.Playbar = (function() {
                     #spicetify-update ol > li {
                         list-style-type: decimal;
                     }
+                    .spicetify-update-space {
+                        margin-bottom: 25px;
+                    }
+                    .spicetify-update-little-space {
+                        margin-bottom: 8px;
+                    }
                 </style>
+                <p>Update Spicetify to receive new features and bug fixes.</p>
+                <div class="spicetify-update-space"></div>
```

Comment:
Might I suggest just adding a class or css to the tags needing bottom margin, instead of adding new DOM elements? 
e.g.
```html
<p class="spicetify-update-space">Update Spicetify to receive new features and bug fixes.</p>
```